src/ui/profile_preferences.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>718</width>
-    <height>661</height>
+    <width>777</width>
+    <height>761</height>
```

Comment:
I think the width was to get the tabs to fit, and the height was probably a typo.